test/resources/expectedOutputs/dependency-just-rdf4j/src/main/java/com/inrupt/generated/vocab/lit/test/SCHEMA_INRUPT_EXT.java

```diff
@@ -1,9 +1,15 @@
 /**
- * Generated by the artifact generator [@inrupt/artifact-generator], version [1.0.0]
+ * Generated by the artifact generator [@inrupt/artifact-generator], version [2.0.0]
  * as part of artifact: [generated-vocab-TEST], version: [0.1.5-SNAPSHOT]
- * on 'Friday, August 20, 2021 10:39 AM'.
+ * on 'Sunday, May 15, 2022 12:05 AM'.
```

Comment:
As a future item, we should find a way to prevent these from being updated with a new date each time.

Reply:
Yeah. This doesn't matter to the test code (as it skips over the initial comment block), but I think you mean that it clutters up code reviews. I've never really liked the way the tests currently compare assert against the entire generated code (as a single string), but yeah, if we ever fix that issue, then maybe this niggly issue will go away too...
Cheers.